src/axom/quest/examples/quest_candidates_example.cpp

```diff
@@ -434,6 +446,7 @@ template <typename ExecSpace>
 axom::Array<IndexPair> findCandidatesBVH(const HexMesh& insertMesh,
                                          const HexMesh& queryMesh)
 {
+  AXOM_ANNOTATE_BEGIN("initializing BVH");
```

Comment:
Minor: Would it make sense to remove the explicit timers now that we have caliper? 

Having both will cause the outer wrapper to include timings for the inner one, and in this case, the caliper timings will include the SLIC formatting and logging times.